e2e/app/element/base-element.ts

```diff
@@ -172,39 +164,43 @@ export default class BaseElement {
 
   /**
    * Clear existing value in textbox then type new text value.
-   * @param textValue The text string.
+   * @param newValue The text string.
    * @param options The typing options.
    */
-  async type(textValue: string, options: { delay?: number } = {}): Promise<this> {
+  async type(newValue: string, options: { delay?: number } = {}): Promise<this> {
     const { delay = 10 } = options;
 
     const clearAndType = async (txt: string): Promise<string> => {
-      await this.clear();
+      await waitForFn(async () => await this.isVisible());
+      const texts = await this.getProperty<string>('value');
```

Comment:
grammar nit: this should probably be `text`, not `texts`, unless I misunderstand what this is doing